AutomatedTesting/Gem/PythonTests/assetpipeline/asset_processor_tests/asset_processor_gui_tests.py

```diff
@@ -350,16 +350,15 @@ def test_APStopTimesOut_ExceptionThrown(self, ap_setup_fixture, asset_processor)
         5. Verify that Asset Processor times out and returns the expected error
         """
 
-        asset_processor.create_temp_asset_root()
         asset_processor.start()
 
         # Copy in some assets, so that the AP will be busy when the stop command is called.
         asset_processor.prepare_test_environment(ap_setup_fixture["tests_dir"], "TimeOutTest")
```

Comment:
I think you might need to leave the create_temp_asset_root call here and set prepare_test_environment to use the existing one, because otherwise it will still create a new one which AP is not aware of (since its already started)